docs/plugins/event-list.md

```diff
@@ -0,0 +1,31 @@
+---
+id: EventList
+title: List of Plugin Events
+sidebar_label: List of Plugin Events
+slug: /plugins/EventList
+---
+
+
+# List of Plugin Events
+
+- [Create Server](plugins\plugin-events\create-server.md)
+- [Create Server Authorization](plugins\plugin-events\create-server-auth.md)
+- [Create Server Docker Compose](plugins\plugin-events\create-server-docker-compose.md)
+- [Create Server Dot Env](plugins\plugin-events\create-server-dot-env.md)
+- [Create Entity Service](plugins\plugin-events\create-entity-service.md)
+  - [Create Entity Service Base](plugins\plugin-events\create-entity-service-base.md)
+  - [Create Entity Controller](plugins\plugin-events\create-entity-controller.md)
+  - [Create Entity Controller Base](plugins\plugin-events\create-entity-controller-base.md)
```

Comment:
missing the CreateEntityControllerSpec

Reply:
also missing: CreateServerPackageJson and CreateAdminUIPackageJson
I think you can add those events after Create Server Docker Compose